spec/features/locations/edit_spec.rb

```diff
@@ -47,5 +47,17 @@
       expect(find('h1')).to have_content('Main site 1')
       expect(find('.govuk-error-summary')).to have_content('Name is missing')
     end
+
+    scenario 'displays the old location name when it is changed' do
+      visit edit_provider_site_path(provider_attributes[:provider_code], site.id)
+
+      expect(find('h1')).to have_content('Main site 1')
+
+      fill_in("Name", with: "Main site 2")
+      click_on('Publish changes')
+
+      expect(find('h1')).to have_content('Main site 1')
+      expect(find('.govuk-error-summary')).to have_content('Name is missing')
```

Comment:
Name is missing?

Reply:
Yes

Reply:
https://github.com/DFE-Digital/manage-courses-frontend/blob/master/app/controllers/sites_controller.rb#L56